apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -154,7 +154,11 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
         }
         return Promise.resolve(true)
     }
-
+    const validateDate = (str) =>  {
+        if (!str) return null
+        const djs = dayjs(String(str))
+        return djs.isValid() && djs.toISOString() || null
```

Comment:
I'd rather do djs.isValid() ?? djs.toISOString() : null, seems to be more obvious